sample-app/Jenkinsfile

```diff
@@ -19,20 +19,12 @@ node {
   switch (env.BRANCH_NAME) {
     // Roll out to staging
     case "staging":
-        // Change deployed image in staging to the one we just built
-        sh("sed -i.bak 's#gcr.io/cloud-solutions-images/gceme:1.0.0#${imageTag}#' ./k8s/staging/*.yaml")
-        sh("kubectl --namespace=production apply -f k8s/services/")
-        sh("kubectl --namespace=production apply -f k8s/staging/")
-        sh("echo http://`kubectl --namespace=production get service/${feSvcName} --output=json | jq -r '.status.loadBalancer.ingress[0].ip'` > ${feSvcName}")
+        sh("kubectl --namespace=staging set image deployments/${appName} ${appName}=${imageTag}"
```

Comment:
Correct, this tutorial takes a single service and deploys 2 matching deployments behind it. I opened #11 a while back and have been remiss in making that change. I will prioritize it.